docs/asciidoc/static/getting-started-with-logstash.asciidoc

```diff
@@ -1,14 +1,12 @@
 == Getting Started with Logstash
 
-[float]
-=== Introduction
-Logstash is a tool for receiving, processing and outputting logs. All kinds of logs. System logs, webserver logs, error logs, application logs and just about anything you can throw at it. Sounds great, eh?
+Logstash is a tool for receiving, processing and outputting logs. All kinds of logs. System logs, webserver logs, error logs, application logs, and just about anything you can throw at it. Sounds great, eh?
 
-Using Elasticsearch as a backend datastore, and kibana as a frontend reporting tool, Logstash acts as the workhorse, creating a powerful pipeline for storing, querying and analyzing your logs. With an arsenal of built-in inputs, filters, codecs and outputs, you can harness some powerful functionality with a small amount of effort. So, let's get started!
+Logstash provides a powerful pipeline for storing, querying, and analyzing your logs. When using Elasticsearch as a backend data store and Kibana as a frontend reporting tool, Logstash acts as the workhorse.  It includes an arsenal of built-in inputs, filters, codecs, and outputs, enabling you to harness some powerful functionality with a small amount of effort. So, let's get started!
 
 [float]
 ==== Prerequisite: Java
-The only prerequisite required by Logstash is a Java runtime. You can check that you have it installed by running the  command `java -version` in your shell. Here's something similar to what you might see:
+The only prerequisite for installing Logstash is a Java runtime. You can verify that you have it installed by running the  command `java -version` in your shell. Here's something similar to what you might see:
```

Comment:
True. Do we want to be more specific about what the minimum version of Java is? The ES guide is similarly vague,  but the reference explictly calls out Java 7 as the minimum version. http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/setup.html#jvm-version

Does it make sense to be consistent about the versions we are recommending?